flocker/node/agents/gce.py

```diff
@@ -822,6 +724,249 @@ def _stop_node(self, node_id):
 
         :param unicode node_id: The compute_instance_id of the node to stop.
         """
+        self._volume_manager.stop_node(node_id)
+
+
+class IGCEVolumeManager(Interface):
```

Comment:
Heh, originally that was named AtomicOperations, and then I changed to VolumeManager on request from Richard because that maps to what we call a similar object in the Cinder driver. But your right, this has more than just volume operations. Renaming to just plain old GCEOperations.